imports.go

```diff
@@ -140,3 +172,23 @@ func isOwnPackage(path, cwd string) bool {
 	}
 	return false
 }
+
+func unique(input []string) []string {
+	u := make([]string, 0, len(input))
+	m := make(map[string]bool)
```

Comment:
please do not use one-symbol variables, it is more clear when name of variable describes what the purpose (or meaning) of variable. 
u -> result
m -> hash or table